lib/dnc/dn.rb

```diff
@@ -26,7 +26,7 @@ class DN
   # @return [DN]
   def initialize(opts = {})
     @dn_string = opts[:dn_string]
-    fail 'dnc: dn_string parameter is **required**' if dn_string.nil?
```

Comment:
It's funny, when I wrote this code Rubocop was still saying `fail` was preferred over `raise`. I'm sad to see it go :(